### DISCUSSION TITLE
Prototype trains

### DISCUSSION THREAD

**Discussion by EightonEight** [KoreanGRF/KoreanTrainSet#302] [건의(Suggestions)]
만약 해무(#98)가 추가된다면 형평성에 맞게 다른 시제차들도 같이 추가하는게 좋을것 같습니다.

HSR-350X (High Speed Rail - 350km/h eXperiment)
![image](https://user-images.githubusercontent.com/77984422/113096979-25ca4d80-9231-11eb-86cd-917a0a1fca9b.png)
<https://blog.naver.com/hunza7/221092537406>
<https://cafe.daum.net/kicha/ANj/23613?svc=cafeapi>

TTX - Tilting Train Express
![image](https://user-images.githubusercontent.com/77984422/113097114-5d38fa00-9231-11eb-97ad-4bab54f70635.png)
<https://ko.wikipedia.org/wiki/%ED%95%9C%EA%B5%AD%ED%98%95_%ED%8B%B8%ED%8C%85%EC%97%B4%EC%B0%A8>
<https://hosii.info/223>

AUTS - Advanced Urban Transit System
![image](https://user-images.githubusercontent.com/77984422/113098073-f1f02780-9232-11eb-99e6-60ac413ac378.png)
<https://blog.naver.com/ianhan/220322890750>
<https://www.krri.re.kr/_prog/brochure_bd/?site_dvs_cd=kr&menu_dvs_cd=060705&func_dvs_cd=brochure_bd&&GotoPage=2>

한국형 표준전동차
![image](https://user-images.githubusercontent.com/77984422/113098240-324fa580-9233-11eb-9acb-8bfa8b7fcab6.png)
![image](https://user-images.githubusercontent.com/77984422/113098249-367bc300-9233-11eb-94a6-bc34d4779e35.png)
<https://blog.naver.com/krriblog/220729889002>
<https://blog.naver.com/ianhan/120028952702>

**Comment by SerpensNebula** [comment]
개인적으론 다른 실험 차량은 추가하지 않더라도 해무는 최고속도가 상당히 높게 나온 상징성이 있어서 일단 추가해두고, 나중에 해무 스펙으로 실차량이 나오면 그걸 따라 해무를 수정하는 게 나을 것 같다고 생각합니다.

다만 형평성이 문제가 된다면 그냥 추가하지 않거나, 전부 추가하고 2cc trains 처럼 따로 on off 할 수 있는 옵션을 넣는 것도 좋을 것 같습니다.

**Comment by kiwitreekor** [comment]
> 개인적으론 다른 실험 차량은 추가하지 않더라도 해무는 최고속도가 상당히 높게 나온 상징성이 있어서 일단 추가해두고, 나중에 해무 스펙으로 실차량이 나오면 그걸 따라 해무를 수정하는 게 나을 것 같다고 생각합니다.
> 
> 다만 형평성이 문제가 된다면 그냥 추가하지 않거나, 전부 추가하고 2cc trains 처럼 따로 on off 할 수 있는 옵션을 넣는 것도 좋을 것 같습니다.

해무기반 실차가 EMU-250/EMU-320이기 때문에 동일 스펙으로 실차가 나올 가능성은 매우 낮습니다. 전 만약 추가한다면 on/off 방식으로 가는 게 맞다고 봅니다.

**Comment by opentrain** [comment]
이거 만들게 된다면 반영가능성이 있을지 모르겠네요

**Comment by EightonEight** [comment]
### 스프라이트(안) ###
![HSR350X](https://user-images.githubusercontent.com/77984422/151088804-d1ee28d2-4d1c-4e6c-b818-a4d734b56f04.png)
![HSR350X (2)](https://user-images.githubusercontent.com/77984422/151095658-4d9f5685-fc2b-4161-897e-b6ab1aa81004.png)
실제 시험용 열차는 7량 1편성이나, 양산 시 11량/20량 1편성 운용을 전제로 개발되었습니다.
```
7량: PC1 - TM1 - TT2 - TT3 - TT4 - TM5 - PC2 
11량: PC1 - TM1 - TT2 - TT3 - TT3 - TT3 - TT4 - TT4 - TT4 - TM5 - PC2 
20량: PC1 - TM1 - TT2 - TT3 - TT3 - TT3 - TT3 - TT3 - TT3 - TM5 - TM5 - TT4 - TT4 - TT4 - TT4 - TT4 - TT4 - TT4 - TM1 - PC2 
```

**Vehicle specification**
| Name      | Code            | Speed             | Speed (designed) | Capacity | Power            | Weight | Introduction |
|-----------|---------------|----------------|---------------------|-----------|----------------|---------|---------------|
| HSR350X | ``HSR350X`` |  **352 km/h** | **385 km/h**        |                | **13200kW** | 68t       | **2002**       |

1호차(TM1): **회의실 (56석)**
2호차(TT2): **특실 26석**
3호차(TT3): **검측실 (60석)**
4호차(TT4): **일반실 56석**
5호차(TM5): **검측실 (28석)**

**References**
* <https://cafe.naver.com/hkct/52119>
* <https://youtu.be/_v4EkxAqywM>
* <https://blog.naver.com/hunza7/221092537406>
* <https://youtu.be/DEJGVMyTljA>
* [한국철도기술연구원, 「Prototype Test Train HSR 350x」, 2006](https://www.krri.re.kr/_prog/brochure_bd/?site_dvs_cd=kr&menu_dvs_cd=060705)
* SETechnology(주), 「호남고속철도 차량 시스템 설계에 관한 연구」, 2001

---
![TTX](https://user-images.githubusercontent.com/77984422/171637799-60616ee2-9def-40ec-8e3f-2fd984218cd3.png)
```
6량: MCP1 - M1 - T1 - T2 - M2 - MCP2
```

**Vehicle specification**
| Name | Code   | Speed             | Speed (designed) | Capacity | Power            | Weight | Introduction |
|--------|--------|-----------------|---------------------|-----------|---------------|---------|---------------|
| TTX     | ``TTX`` |  **180 km/h** | **200 km/h**        | **29**     | **4000kW** | 60t       | **2007**       |

1호차(MCP1): **특실 29석**
2호차(M2): **일반실 56석**
3호차(T3): **일반실 56석**
4호차(T4): **일반실 52석**
5호차(M5): **일반실 56석**
6호차(MCP6): **특실 29석**

**References**
* <https://blog.naver.com/sburst/50039599575>
* <https://cafe.naver.com/korailslr/20356>
* <https://blog.naver.com/zxc4470185/220461995070>
* <https://youtu.be/WwjZZqwbaGI>
* <https://youtu.be/mkyXzDfIsWg>
* 철도연 기존철도기술개발사업단장, 「한국형 틸팅열차의 개발 및 성과」, 한국철도기술, 2007.01

---
![AUTS](https://user-images.githubusercontent.com/77984422/151950416-3b71fac5-5c72-4932-987f-00f78109b01f.png)
```
6량: MC1 - TP1 - M - T - TP2 - MC2
```

**Vehicle specification**
| Name | Code     | Speed             | Speed (designed)   | Capacity | Power           | Weight | Introduction |
|--------|----------|-----------------|---------------------|-----------|---------------|---------|---------------|
| AUTS  | ``AUTS`` | **110 km/h**  | **130 km/h**         | 100         | **2400kW** | **36t**  | **2010**       |

1호차(MC1): **도심혼잡형 135명**
2호차(TP2): **교외형 143명**
3호차(M3): **첨단형 143명**
4호차(T4): **첨단형 143명**
5호차(TP5): **교외형 143명**
6호차(MC6): **도심혼잡형 135명**

**References**
* <https://blog.naver.com/ianhan/220322890750>
* <https://blog.naver.com/digmon2001/220444585426>
* <https://youtu.be/H-yic8qK81g>
* <https://youtu.be/AW-8dG4jpic>
* [한국철도기술연구원, 「차세대 첨단 도시철도시스템 기술개발」, 2011](https://www.krri.re.kr/_prog/brochure_bd/?site_dvs_cd=kr&menu_dvs_cd=060705)
* 한국철도기술연구원, 「차세대 첨단 도시철도시스템 기술개발사업」, 2011.10

---
![STANDARD_KEMU](https://user-images.githubusercontent.com/77984422/151950449-13e2363f-bf3a-4883-af64-523eb5cec08d.png)
위 - 롱시트(지하철)형
아래 - 크로스시트(일반열차)형
```
4량: TC1 - M' - M - TC2
```

**Vehicle specification**
| Name                    | Code                          | Speed             | Speed (designed)   | Capacity | Power           | Weight | Introduction |
|---------------------|------------------------|-----------------|----------------------|-----------|--------------|----------|---------------|
| STANDARD KEMU | ``STANDARD_KEMU`` | **100 km/h**  | **100 km/h**         | 100         | **1600kW** | **32t**  | **1999**       |

**References**
* <https://blog.naver.com/ianhan/120130499957>
* <http://2427junction.com/koreareportrc.html>
* 한국철도기술연구원 등, 「도시철도차량의 시스템 경량화 기술에 관한 연구」, 2011

---
![K-HYDROGEN](https://user-images.githubusercontent.com/77984422/151986661-d47a0b7a-b440-49c4-b50c-411df99fecfa.png)
실제 열차는 2량 1편성이나, 게임 플레이 편의를 위해 가상의 중간 객차가 포함되었습니다.

**Vehicle specification**
| Name              | Code                   | Speed            | Speed (designed) | Capacity | Power         | Weight  | Introduction |
|-----------------|--------------------|---------------|---------------------|----------|--------------|----------|---------------|
| K-HYDROGEN | ``K-HYDROGEN`` | **110 km/h** | **121 km/h**       | **58**    | **1300kW** | **45t** | **2021**       |

1호차: **일반실 58석**
2호차: **일반실 58석**

**References**
* <https://www.redaily.co.kr/sub_read.html?uid=2629>
* <https://gall.dcinside.com/mgallery/board/view/?id=monorail&no=226587>
* <https://youtu.be/BrhgAWj4etY?t=40>
* <https://youtu.be/mg-U8oJ8tOw>
* <https://youtu.be/mMMzvWTdp-w>
* <https://youtu.be/0N_7yybLBZk?t=415>

---

**Comment by EightonEight** [comment]
[MEGB-NewGRF/KoreanTrainSet_Plus#47](https://github.com/MEGB-NewGRF/KoreanTrainSet_Plus/issues/47), [MEGB-NewGRF/KoreanTrainSet_Plus#48](https://github.com/MEGB-NewGRF/KoreanTrainSet_Plus/issues/48)로 이관합니다.
